app/views/departments/edit.html.slim

```diff
@@ -4,6 +4,5 @@
 
 == render 'form'
 
-.row
-  .col-xs-12
-    = link_to t(:back), departments_path, class: 'btn btn-default'
\ No newline at end of file
+= form_navigation_btn :show
```

Comment:
you probably meant the opposite - when I check out at https://github.com/panter/aoz-003/commit/02712307e311c5c8913a01cb683647d6661b816e - and I remember deleting it for the related files.
but anyway - do we need/does it make sense to have the show button in this view or what do you think?